api/space_api.py

```diff
@@ -52,6 +52,22 @@ class Meta:
         }
 
 
+class DistrictSerializer(TranslatedModelSerializer):
+    class Meta:
+        model = District
+        fields = ["id", "district_type", "name", "parent"]
```

Comment:
If parent is id field, should be named parent_id in the serializer according to contributing docs.

Reply:
maybe same for district_type?